index.html

```diff
@@ -143,46 +142,46 @@ <h1 id="title">[PROPOSED] Verifiable Credentials Working Group Charter</h1>
         </table>
       </section>
 
-      <!-- MARK: Scope -->
-      <section id="scope" class="scope">
-        <h2>Scope</h2>
+      <!-- MARK: Background -->
+      <section id="background" class="background">
+        <h2>Motivation and Background</h2>
         <p>
-          Building on the experience gained through implementation, deployment and usage
-          of Verifiable Credentials (VCs), this Working Group will extend Verifiable
-          Credential foundations with new standardized technologies to improve the use of
-          this technology on the Web.
+          In the past few years the Working Group has developed a family of specifications in the area of Verifiable Credentials. 
+          They provide a mechanism to express "credentials" 
+          (driver's licenses, university degrees, government-issued residence permits, etc.) in a digital 
+          format on the Web that is cryptographically secure, privacy respecting, and machine-verifiable.
+          These technologies are being gradually adopted by various institutions, governmental or otherwise, in different 
```

Comment:
```suggestion
          (driver's licenses, university degrees, government-issued residence permits, etc.) on the Web in a digital 
          format that is cryptographically secure, privacy respecting, and machine-verifiable.
          These technologies are being gradually adopted by various institutions, governmental and otherwise, in different 
```